src/components/TextInput/BaseTextInput/index.native.tsx

```diff
@@ -154,7 +186,7 @@ function BaseTextInput(props) {
             isFocused ||
             // If the text has been supplied by Chrome autofill, the value state is not synced with the value
             // as Chrome doesn't trigger a change event. When there is autofill text, keep the label activated.
-            isInputAutoFilled(input.current)
+            isInputAutoFilled(input.current as unknown as Element)
```

Comment:
You can update `input` to be 
```
const input = useRef<TextInput & HTMLElement>(null);
```
`isInputAutoFilled` to be 
```
export default function isInputAutoFilled(input: (TextInput & HTMLElement) | null): boolean {
```

```suggestion
            isInputAutoFilled(input.current);
```